vscode-antimony/src/server/stibium/stibium/parse.py

```diff
@@ -75,8 +75,6 @@ def parse(self, text: str, recoverable=False) -> FileNode:
         tree: Tree
         root_puppet = get_puppet(self.parser, 'root', text)
         tree = self._parse_with_puppet(root_puppet, recoverable, text)
-        vscode_logger.info("Parse tree:")

```

Comment:
Add this back for debugging purposes, I'll comment these out when deploying to prod.